dask/dataframe/tseries/tests/test_resample.py

```diff
@@ -38,6 +38,27 @@ def test_series_resample(obj, method, npartitions, freq, closed, label):
     assert expected.index[-1] == divisions[-1]
 
 
+def test_resample_agg():
+    index = pd.date_range('1-1-2000', '2-15-2000', freq='h')
+    ps = pd.Series(range(len(index)), index=index)
+    ds = dd.from_pandas(ps, npartitions=2)
+    result = ds.resample('10min').agg('mean')
+    expected = ps.resample('10min').agg('mean')
+    assert_eq(result, expected, check_dtype=False)
+    result = ds.resample('10min').agg(['mean', 'min'])
+    expected = ps.resample('10min').agg(['mean', 'min'])
+    assert_eq(result, expected, check_dtype=False)
+
+
+def test_resample_agg_passes_kwargs():
+    index = pd.date_range('1-1-2000', '2-15-2000', freq='h')
+    ps = pd.Series(range(len(index)), index=index)
+    ds = dd.from_pandas(ps, npartitions=2)
+    def foo(series, *args, bar=1):
+        return [bar] * len(series)
+    res = ds.resample('10min').agg(foo, bar=2)
```

Comment:
@TomAugspurger can't get this mock test to work. Not sure the correct syntax for an aggregate function. Tried a variety of signatures.

Reply:
I would avoid functions that return sequences. Does it work if you just apply a function like `len(x) * bar

And does it fail on the dask side or the pandas side?

Reply:
I get a dask error if i try to compute: 

```
TypeError: foo() missing 1 required positional argument: 'series'
```

The odd part is that it gets past the step where it tries to determine the meta here: https://github.com/dask/dask/pull/3202/files#diff-394a2aeb5f331b788b4adb6242f9091aR125

Reply:
This is because having the tuple `(foo,)` in the task graph gets evaluated as `foo()`.  I've changed this to `[foo]` and pushed.

Reply:
I've also screwed with the test a bit.  Things fail around the midpoint of the resample.

Reply:
@mrocklin thanks, with your fix and choosing a proper sample freq the test passes.